src/main/java/org/swisspush/reststorage/RedisRestStorageRunner.java

```diff
@@ -0,0 +1,23 @@
+package org.swisspush.reststorage;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import org.slf4j.LoggerFactory;
+import org.swisspush.reststorage.util.ModuleConfiguration;
+
+/**
+ * Runner class to start rest-storage in standalone mode with a redis based storage
+ *
+ * @author https://github.com/mcweba [Marc-Andre Weber]
+ */
+public class RedisRestStorageRunner {
+
+    public static void main(String[] args) {
+        ModuleConfiguration modConfig = new ModuleConfiguration()
+                .storageType(ModuleConfiguration.StorageType.redis)
+                .redisReconnectAttempts(-1);
+
+        Vertx.vertx().deployVerticle("org.swisspush.reststorage.RestStorageMod", new DeploymentOptions().setConfig(modConfig.asJsonObject()), event ->
+                LoggerFactory.getLogger(RestStorageMod.class).info("rest-storage started"));
```

Comment:
```java
    LoggerFactory.getLogger(RestStorageMod.class).info("rest-storage started"));
```
Should we really use a logger of a foreign class? Intuitively I would expect "*RedisRestStorageRunner.class*" here.